examples/mne_connectivity_classification.py

```diff
@@ -0,0 +1,202 @@
+"""
+.. _ex-var-classification:
+
+===============================================
+Classify sleep stages by computing connectivity
```

Comment:
```suggestion
Classify sleep stages using connectivity measures
```
I would argue that it is not the connectivity that does the actual classification :)